CoastalResilienceNetwork/GeositeFramework
# Adjust jQuery library imports for Plugins

### Overview

This PR updates the jQuery library imports for `chosen` and `jquery-ui-slider-pips` to declare their dependencies explicitly in `Index.cshtml`. This in turn forces them to load in the correct order when imported into PluginBase.

As a consequence of these changes, we can revert the adjustments made to the `pips` module in #906 and the explicit function call, and also replace the patched version of `pips` with a minified version.

The second commit here fixes a typo in one of the `pips` options.

Connects #910 

### Screenshots

<img width="1218" alt="screen shot 2017-03-10 at 11 33 36 am" src="https://cloud.githubusercontent.com/assets/4165523/23803734/73ec3772-0585-11e7-997c-5ad8d10b4b07.png">

### Testing
 * remove all plugins which are not part of the base framework on your local
 * rebuild this branch in VS & view it in the browser with the console open
 * open the sample plugin and verify that the new UI elements display & work correctly: the slider should have pips, the accordion menu should work, and the select box should have chosen's style
 * add the regional planning and benefits explorer plugins, then rebuild this and open the sample plugin again. Verify that the sample plugin still has the correct appearance and behavior
